src/minerfund.cpp

```diff
@@ -41,6 +56,15 @@ std::vector<CTxOut> GetMinerFundRequiredOutputs(const Consensus::Params &params,
         // Tobias
         BuildOutput("qzafzyamh8rgszacw73gg2vly8gnxe099qx6fqrdfd", shareAmount),
         // Burn the remaining shares
-        BurnOutput(10 * shareAmount),
+        BuildWinnerOutput(1, shareAmount, epochBlockHash),
```

Comment:
For the testnet, can you burn 2 outputs and make the remaining 8 random? This will be more similar to the behavior of the mainnet.

Reply:
?

Reply:
Mainnet will have 5 fixed and 8 random outputs. Atm, we only have 3 fixed outputs. Let's burn two and make 8 random.

Right now it's 3 fixed and 10 random.